controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -229,6 +232,30 @@ func (in *KubeadmControlPlane) validateCommon() (allErrs field.ErrorList) {
 	return allErrs
 }
 
+func (in *KubeadmControlPlane) validateEtcd(prev *KubeadmControlPlane) (allErrs field.ErrorList) {
+	if in.Spec.KubeadmConfigSpec.InitConfiguration.Etcd.External != nil && prev.Spec.KubeadmConfigSpec.InitConfiguration.Etcd.Local != nil {
+		allErrs = append(
+			allErrs,
+			field.Forbidden(
+				field.NewPath("spec", "kubeadmConfigSpec", "initConfiguration", "etcd", "local"),
```

Comment:
Ah, this one should be "external" instead of "local".

The reason is that this field is the thing that gets bubbled up to the user and the field on the `in` type is the one that the user tried to change. So if they change `external` but then we tell them `local` is forbidden that's a pretty confusing error message

Reply:
makes sense